pkg/kube/cluster-init.sh

```diff
@@ -891,6 +892,10 @@ else
                 if [ ! -f /var/lib/node-labels-initialized ]; then
                         reapply_node_labels
                 fi
+                if ! external_boot_image_import; then
+                        continue
```

Comment:
It is quite hard to see that this is actually in a (gigantic) while loop and that the continue makes sense. Part of this is the size of the while loop, and part of it is that it is not indented! (latter leaves some indentation oddities at the end of the file)
I'm very surprised that shellcheck doesn't complain about the indentation, but can we 1) either fix the indentation so that the body of the do while is corrrectly indented, or 2) refactor this to make it more readable and maintainable?
If so, please do that as a separate commit or separate PR.

Reply:
Sure will submit a separate PR with some code refactoring and indentation